python/hail/expr/expression.py

```diff
@@ -3,6 +3,7 @@
 from hail.expr.ast import *
 from hail.expr.types import *
 from hail.utils.java import *
+from hail.utils.misc import plural
 from hail.utils.linkedlist import LinkedList
```

Comment:
Did you consider adding the method name checking to the Expression types?

Reply:
ah, good point. I can totally add this to Struct/StructExpression. Can I do that in a future PR?